doc: Add some doc for the local konnector install (#3927)
cozy/cozy-stack: docs/konnectors-dev.md

[Table of contents](README.md#table-of-contents)

> # Develop a konnector

Reviewer: "Running a konnector locally" instead?

> ## In standalone 

Reviewer: ```suggestion
## In standalone mode
```

> This allows to run a konnector without a cozy-stack available. This is probably

Author: I remove the doc then

the best option for the early development. 

You can find a detailed explanation with a video inside the [konnector getting started](https://docs.cozy.io/en/tutorials/konnector/getting-started/)
page.

> ## Run in local and save in a stack

Reviewer: ```suggestion
## Locally but linked to a Cozy
```

> This method requires a working cozy-stack. It will allows you to run the konnector

Reviewer: ```suggestion
This method requires a working cozy-stack. It will allow you to run the konnector
```

> outside the cozy-stack but saves the data inside the cozy-stack. This can be usefull

Reviewer: ```suggestion
outside the cozy-stack but saves the data inside the cozy-stack. This can be useful
```

> when your konnector is not available inside the registry but you still want to save

Reviewer: ```suggestion
when your konnector is not available in one of the registries but you still want to save
```

Reviewer 2: @Peltoche it looks like you have missed some suggestions made by Erwan

> you data in your real cozy.

Reviewer: ```suggestion
you data in your real Cozy.
```

You can find a detailed explanation inside the [linking your konnector to a cozy](https://docs.cozy.io/en/tutorials/konnector/save-data/#linking-your-connector-to-a-cozy-dev-mode)
page.

> ## Install you konnector without the registry

Reviewer: ```suggestion
## Installed from a local directory
```

This method requires a working cozy-stack with the admin permissions and the cozy-stack 
CLI. It will allows you to install you konnector exactly like the real registry. This 
can be a good way to check locally if everything works well in local before publishing 
to the registry.

### Install you konnector

First build you konnector:

```
yarn build
```

Start your cozy-stack server with the `--dev` flag. If you have the cozy-stack source code
locally you can run this following command for example:
```
go run . serve --dev --config ~/.config/cozy/config.yaml --mailhog --fs-url=file://localhost${PWD}/storage --konnectors-cmd ${PWD}/scripts/konnector-dev-run.sh
```

> You will need to have a valid configuration file.

Then you just have to run:

```
cozy-stack konnectors install <slug> files://<konnector folder absolute path>/build
```

> Be careful to point the `/build` folder inside your app!

### Run you konnector

> Once connected to your cozy instance (probably `http://cozy.localhost:8080`) you should 

Reviewer: It's not a big deal but we always start `Cozy` with an uppercase letter.
